auditbeat/docs/setting-up-running.asciidoc

```diff
@@ -5,7 +5,11 @@
 /////
 
 [[setting-up-and-running]]
-== Setting up and running {beatname_uc}
+== Set up and run {beatname_uc}
+
+++++
+<titleabbrev>Set up and run</titleabbrev>
+++++
 
 Before reading this section, see the
 <<{beatname_lc}-getting-started,getting started documentation>> for basic
```

Comment:
Match the new heading?

```suggestion
<<{beatname_lc}-getting-started,get started documentation>> for basic
```

Reply:
Actually the text here is descriptive, so it doesn't need to exactly match the target topic.